l10n_uy_ux/models/account_move.py

```diff
@@ -52,6 +53,17 @@ def _l10n_uy_edi_check_move(self):
 
         return errors
 
+    def _post(self, soft=True):
+        # EXTENDS l10n_uy_edi
```

Comment:
agregar doscstring explicando el porque  extendemos
```suggestion
        # EXTENDS l10n_uy_edi
        """ Extendemos para tras confirmar las factuas hagamos el envio directo a DGI, en caso de que de error entonces devolvemos factura a draft """
```